### DISCUSSION TITLE
TexStodio internal PDF viewer show chapters

### DISCUSSION THREAD

**Discussion by Koushikphy** [texstudio-org/texstudio#3267] [General]
Is there any possibility to the show the chapters/sections of a pdf directly inside the internal pdf viewer of the TexStudio? Presently only the generated pdf is shown in the viewer. But a sidebar with the chapters/sections of the pdf will be helpful for easy navigation.

**Comment by octaeder** [comment]
use pdf's context menu or viewer's `Window` sub menu `Show/Contents`

**Reply by Koushikphy** [reply]
Okay, thats perfect.
